presto-iceberg/src/main/java/io/prestosql/plugin/iceberg/IcebergMetadata.java

```diff
@@ -453,6 +457,12 @@ public ConnectorInsertTableHandle beginInsert(ConnectorSession session, Connecto
                 .collect(toImmutableList())));
     }
 
+    @Override
+    public ColumnHandle getUpdateRowIdColumnHandle(ConnectorSession session, ConnectorTableHandle tableHandle)
+    {
+        return new IcebergColumnHandle(0, "$shard_row_id", BIGINT, Optional.empty());
```

Comment:
We can just call this `$row_id`. The "shard" name is specific to Raptor

Reply:
Changed.